transform/snowflake-dbt/macros/helpers.sql

```diff
@@ -305,7 +306,16 @@ select get_sys_var({{ var_name }})
                  FROM {{ this }} 
                  WHERE {{this}}.received_at <= CURRENT_TIMESTAMP
                  GROUP BY 1
-             ), 
+             ),
+        {%+ elif this.table == 'performance_events' %}
```

Comment:
could we define a static var for 'performance_events' - might be useful for testing as well (have to rename the table to smth like `performance_events_test` in only one place)